zappa/cli.py

```diff
@@ -757,7 +758,8 @@ def deploy(self, source_zip=None):
                 memory_size=self.memory_size,
                 runtime=self.runtime,
                 aws_environment_variables=self.aws_environment_variables,
-                aws_kms_key_arn=self.aws_kms_key_arn
+                aws_kms_key_arn=self.aws_kms_key_arn,
+                layers=self.layers
```

Comment:
You pass this into the create, but you also need to pass it into the self.zappa.update_lambda_configuration function call in this file